pkg/subctl/benchmark/latency.go

```diff
@@ -0,0 +1,69 @@
+package benchmark
+
+import (
+	"fmt"
+
+	"github.com/onsi/gomega"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func StartLatencyTests() {
+	var f *framework.Framework
+
+	gomega.RegisterFailHandler(func(message string, callerSkip ...int) {
```

Comment:
I guess that we will need the same on the benchmark patch. Can you open an PR for it?

Reply:
Sure, I'll open a PR